utils/config/daos_server.yml

```diff
@@ -422,15 +422,15 @@
 #  #
 #  # Optional parameter; set either this option or first_core, but not both.
 #
-#  pinned_numa_node: 1
+#  #pinned_numa_node: 1
```

Comment:
Why double commented?

Reply:
Now I've tightened up the validation, in the test that un-comments daos_server.yml it would fail if this was specified in addition to first_core being set greater than zero.

Reply:
Okay. I'm not a big fan of changing our config file formatting to accommodate unit tests, but it's a small thing and any user will probably do the right thing anyway in the short term. With the two fields incompatible now, we should probably come up with another way of generating our test configs in the longer term.